packages/flame/test/components/component_test.dart

```diff
@@ -344,6 +344,51 @@ void main() {
           expect(wrapper.contains(child), true);
         },
       );
+
```

Comment:
Can we make a group for "onChildrenChanged"?

Reply:
Sure